contracts/cw1/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "cw1"
+version = "0.1.2"
+authors = ["Jan Woźniak <jan@confio.gmbh>"]
+edition = "2021"
+description = "todo"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/sylvia"
+homepage = "https://cosmwasm.com"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+cw-utils = "0.14.0" 
+cw1 = "0.14.0" 
+cw2 = "0.14.0" 
+cosmwasm-std = { version = "1.0.0", features = ["staking"] }
+cw-storage-plus = "0.14.0" 
+schemars = "0.8.1"
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
```

Comment:
Also, I think you should get rid of serde - it is reexposed by Sylvia, and it would be good to use `sylvia::serde` in place of `serde` to have version compatibility guaranteed. Same for `schemars`.

Reply:
As discussed offline this has to stay as sylvia-derive from which interface macro is being used requires it.